src/components/drumMachine.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Button from './button';
+
+const data = [
+  { id: 'snare', letter: 'Q', src: 'https://www.myinstants.com/media/sounds/snare.mp3' },
+  { id: 'bass 1', letter: 'W', src: 'https://www.myinstants.com/media/sounds/bass-drum.mp3' },
+  { id: 'bongo', letter: 'E', src: 'http://tipiwiki.free.fr/snd/Percussion(4e)2.wav' },
+  { id: 'tom tom', letter: 'A', src: 'http://www.denhaku.com/r_box/sr16/sr16tom/loelectm.wav' },
+  { id: 'bass 3', letter: 'S', src: 'http://billor.chsh.chc.edu.tw/sound/bass4.wav' },
+  { id: 'shotgun', letter: 'D', src: 'http://david.guerrero.free.fr/Effects/ShotgunReload.wav' },
+  { id: 'high hat', letter: 'Z', src: 'http://www.denhaku.com/r_box/tr707/closed.wav' },
+  { id: 'drum hit', letter: 'X', src: 'http://www.masterbits.de/sc_docu/sounds1/1TM00013.MP3' },
+  { id: 'laser', letter: 'C', src: 'http://www.sa-matra.net/sounds/starcontrol/Umgah-Backzip.wav'  },
+]
+
+const easterEgg = {
+  id: 'surprise', src: "https://www.myinstants.com/instant/bongo-cat-lets-go-26369/embed/"
+}
+const DrumMachine = () => {
+  return (
+    <div className='drum-machine'>
+      <div className='display'> 
+        {data.map((key,ix) => (
+          <Button text={key.letter} audio={key.src} key={ix}/>
+        ))}
+      </div>
+    </div>
+  )
+}
```

Comment:
Im fairly new to coding and I am learning alot just by looking at yours. Thanks for being here today